### ISSUE TITLE
Document regexes match against a down-cased input

### ISSUE THREAD

**Issue by djgraff209** [Appdynamics/rabbitmq-monitoring-extension#22]
The regular expression matches are executed against a target that has been forced to lower case.

The regular expression matching utilizes java.lang.String.matches(String regex) which is equivalent to java.util.regex.Pattern.matches(regex, input).

This match is NOT case insensitive. The inputs provided are all forced to lower case and should be matched against as such.

**Comment by djgraff209** [comment]
@akshayAppd - I apologize if I was not clear. I was asking for this to be documented - no change to actual behavior.

I have been working with an AppDynamics partner and we discovered this. It was a misunderstanding that required reading the actual source code to determine the behavior.

One of the consequences of this is that if the expressions are incorrectly specified or overly inclusive there will be a pollution of the metric tree within the AppDynamics controller which requires corporate support to resolve.

**Comment by akshayAppd** [comment]
@djgraff209 Yes, we will document this on our extension community page. Thanks for pointing it out.
